packages/v4/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,345 @@
-              Build scalable experiences in the open.
-            </Title>
-            <Text className="pf-v5-c-title pf-m-xl pf-m-white pf-v5-u-mb-md pf-v5-u-mt-lg pf-v5-u-mb-3xl-on-md fadeInUp animated fadeInThree">
-              PatternFly is an open source design system built to drive consistency and unify teams.
-            </Text>
-            <div className="pf-v5-u-display-flex pf-v5-u-justify-content-center pf-v5-u-flex-direction-column pf-v5-u-flex-direction-row-on-md">
+            <Flex
+              direction={{default: "column"}}
+              spaceItems={{default: "spaceItemsLg"}}
+            >
+              <Title
+                className="pf-v5-u-font-weight-bold"
+                headingLevel="h1"
+                size="4xl"
+              >
+                <Text className="pf-v5-u-content-bold">
+                  {" "}
+                  Design and build better product experiences in the
+                  <Text component="span" className="pf-v5-u-primary-color-100">
+                    {" "}
+                    open{" "}
+                  </Text>
+                  with PatternFly
+                </Text>
+              </Title>
+              <Text component={TextVariants.p} className="ws-mdx-p">
+                PatternFly is an open source design system that enables teams to
+                create consistent and scalable enterprise products. PatternFly
+                is sponsored and maintained by Red Hat, but is open to all.
+              </Text>
+            </Flex>
+          </TextContent>
+          <Flex
+            flexWrap={{default: "nowrap"}}
+            spaceItems={{default: "spaceItemsMd"}}
+          >
+            <FlexItem>
+              <Link
+                to="/get-started/design"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start designing"
+              >
+                Start designing
+              </Link>
+            </FlexItem>
+            <FlexItem>
               <Link
-                to="/get-started/about"
-                className="pf-v5-c-button pf4-c-button__cta-primary pf-v5-u-mr-0 pf-v5-u-mb-md pf-v5-u-mb-0-on-md pf-v5-u-mr-xl-on-md fadeIn animated fadeInFour"
-                aria-label="Get started with PatternFly"
+                to="/get-started/develop"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start developing"
               >
-                Get started
+                Start developing
               </Link>
-              <a className="pf-v5-c-button pf4-c-button__cta-outline fadeIn animated fadeInFour" href="//github.com/patternfly" target="_blank">
-                <GithubIcon className="pf-v5-u-mr-sm" />GitHub
+            </FlexItem>
+          </Flex>
+        </Flex>
+        <Flex flex={{sm: "flexNone", md: "flex_3", lg: "flex_3"}}>
```

Comment:
You don't need to redefine this and it's not set to flex by default

```suggestion
        <Flex flex={{md: "flex_3"}}>
```